backend/infrahub/core/validators/attribute/regex.py

```diff
@@ -78,14 +78,14 @@ def name(self) -> str:
         return "attribute.regex.update"
 
     def supports(self, request: SchemaConstraintValidatorRequest) -> bool:
-        return request.constraint_name == self.name
+        return request.constraint_name in (self.name, "attribute.parameters.regex.update")
```

Comment:
It looks like `attribute.parameters.regex.update` should be a global constant. Even if we do not follow much this principle in the codebase, I think we should start doing it